specifyweb/frontend/js_src/lib/components/FormParse/__tests__/cells.test.ts

```diff
@@ -393,10 +409,10 @@ describe('parseFormCell', () => {
       parseFormCell(
         schema.models.CollectionObject,
         strictParseXml(
-          `<cell type="panel" colDef="1px,2px,2px">
+          `<cell type="panel" colDef="1px,2px,2px" initialize="verticalAlign=center;">
```

Comment:
Good job on updating tests to be comprehensive. All looks good.

A small lesson for the future in regard to tests:
It's advisable to keep tests small and not to test the same feature multiple times. Reasons:
1. Smaller tests are easier to maintain (i.e, less code to maintain) and easier to read (i.e, less code to read and to understand)
2. If for example, you rename the verticalAlign attribute or do some other change, if you only had 3 tests testing that feature, then there would be less refactoring work than if you had 9 tests
3. Each new tests means tests run slower, thus redundant tests could be waisting developer's time

Thus, it's very important to have a test for a feature, and in some cases to test that feature comprehensively in different scenarios (vertical align center, left, right), but it's also important not to test the same thing multiple times.

In this case it's not a big deal to test `verticalAlign=center` multiple times, but for some bigger things (i.e automappaer or testing a react component) try to have **comprehensive and succinct** tests.